### DISCUSSION TITLE
2007/07/16/%E9%A2%84%E9%98%B2%E8%BF%91%E8%A7%86%E8%A6%81%E4%BB%8E%E5%A9%B4%E5%84%BF%E6%8A%93%E8%B5%B7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#172] [Announcements]
# 2007/07/16/%E9%A2%84%E9%98%B2%E8%BF%91%E8%A7%86%E8%A6%81%E4%BB%8E%E5%A9%B4%E5%84%BF%E6%8A%93%E8%B5%B7

今天早上看书，看到《心理学》关于视觉的一章了。其中一个非常有用的信息，以前居然从没听说过。这是一项关于婴儿睡眠和近视眼的统计数据。

https://blog.qizhen.xyz/2007/07/16/%E9%A2%84%E9%98%B2%E8%BF%91%E8%A7%86%E8%A6%81%E4%BB%8E%E5%A9%B4%E5%84%BF%E6%8A%93%E8%B5%B7

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Wei
2007年08月14日 14:36 编辑
这篇比较有实际效果 

回复
花
2007年07月29日 11:48 编辑
申请了spaces有段时间了，今天才发现，我的好友列表上有这么一位能人志士！我学计算机的，以后有什么问题，请多多指教哈！ 

回复
Qizhen
2007年07月28日 12:11 编辑
我是一个研发工程师》如果你的网站希望得到NI公司的帮助，你可以试着和NI的市场、销售部门联系一下。

回复
丹
2007年07月27日 18:12 编辑
阮先生，您好！慕名前来您的空间，内容很丰富啊。请问您现在是由NI总部公派到上海负责国内的相关事务吗？目前我工作所在的一个电子行业社区网站有一些问题希望得到您的指教。谢谢！

回复
PhantomKid
2007年07月20日 14:19 编辑
你好 我是一个正在实习的电子工程的学生 但是我公司的instructor让我编个程序 怎么样用vb打开labview阿
你是高手又是前辈 多多指教小弟阿 谢谢了！ 

回复
秋
2007年07月18日 10:08 编辑
呵呵~有意思~
我到现在眼睛很好肯定是小时候我老爸老妈对我照顾有佳~
哈哈~

回复
奕瑋
2007年07月17日 16:32 编辑
多谢阮主提供如此有价值的信息，我以后让儿子关灯睡觉，呵呵。

回复
